srcjs/init_shiny.js

```diff
@@ -128,7 +136,13 @@ function initShiny() {
 
         var type = binding.getType(el);
         var effectiveId = type ? id + ":" + type : id;
-        currentValues[effectiveId] = binding.getValue(el);
+        inputItems[effectiveId] = {
```

Comment:
From discussion, it seems like the behavior we have today would cause immediate=FALSE when the exported bindAll is called. Please follow up to see if indeed immediate=FALSE. In any case we want this to be immediate going forward.